### PR TITLE
Fix keyboard shortcuts from search 

### PR DESCRIPTION
**Pull request by dagheyman**
Related to #222 .

These changes still has the search field as default focus, but allows CTRL/CMD + C to be used to copy the selected credential and re-enables ESC for clearing the search field.

### CODE REVIEW

File: qml/main.qml [hunk 1]
```diff
@@ -326,17 +329,36 @@ ApplicationWindow {
...
+            Keys.onPressed: {
+                if(event.matches(StandardKey.Copy)) {
+                    copyAction.trigger()
+                }
+            }
+
+            function clear() {
+                search.text = ""
+                arrowKeys.forceActiveFocus()
+                deselectCredential()
```
Comment (emlun) [inline qml/main.qml]: Sure about clearing the selection? It's now been fixed to be stable across changing search queries; I feel like it makes more sense to keep the selection where it is (by just deleting this line).

**Comment by emlun** [comment]
This solves #222, but not #226. Should we remove the <kbd>Space</kbd> shortcut for generating codes?

File: qml/main.qml (hunk 1 above)
Comment (dagheyman) [reply qml/main.qml]: Hmm, really? If I'm clearing the search with the ESC key, I wouldn't want any credential to be selected. I think this pattern is pretty common in search fields.

**Comment by dagheyman** [comment]
> Should we remove the Space shortcut for generating codes?

I don't know, I thought about it as well. Right now it works when the search field doesn't have focus. Do you mean removing just the keyboard-shortcut or removing the action (generate but don't copy) completely?

My guess would be that there is some users that wants to be able to generate codes without adding the OTP to the clipboard.

**Comment by emlun** [comment]
Oh right, it's a different action. I meant just removing the keyboard shortcut. In that case perhaps we should revert to not focusing the search field by default.

File: qml/main.qml (hunk 1 above)
Comment (emlun) [reply qml/main.qml]: Hm, ok. I think I'd expect that if I type "github" into the search box and get my GitHub credential selected, it'll remain selected after I hit <kbd>Esc</kbd>. But it's no big deal, I'm fine with this as is if you prefer it that way.

**Comment by dagheyman** [comment]
> Oh right, it's a different action. I meant just removing the keyboard shortcut. In that case perhaps we should revert to not focusing the search field by default.

Let's leave it as it is for now. We'll keep the space-shortcut and keep search focused by default. The shortcut will not work when the search has focus (a space will be printed instead), but it will work after tabbing out or selecting a credential with the mouse. I think searching and filtering for credentials is a more common use case than generating a code without copying it, so it make sense to have that require less steps.

**Pull request merged**